src/plugins/couchbase-lite.ts

```diff
@@ -0,0 +1,35 @@
+import { Plugin, Cordova } from './plugin';
+
+/**
+ * @name Couchbase Lite
+ * @description
+ * Plugin to install Couchbase Lite in your PhoneGap app on iOS or Android
+ *
+ * @usage
+ * ```
+ * import { CouchbaseLite } from 'ionic-native';
+ *
+ * CouchbaseLite.getURL()
+ *   .then((url: any) => console.log(url))
+ *   .catch((error: any) => console.log(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'Couchbase Lite',
+  plugin: '',
```

Comment:
the `plugin` property should be `https://github.com/couchbaselabs/Couchbase-Lite-PhoneGap-Plugin` since they don't have an NPM package pubished